samples/gen-go-db/server/db/interface.go

```diff
@@ -1135,12 +1146,6 @@ type GetThingWithMatchingKeyssByAssocTypeIDAndCreatedBearInput struct {
 	Limit *int64
 }
 
-// CreatedBear struct.
-type CreatedBear struct {
-	Created strfmt.DateTime
-	Bear    string
-}
-
```

Comment:
I was concerned for a second when I saw that this bear struct was deleted, but I see that it's just moved down 😌 .

Reply:
worry not 😌 😌 😌